storage/innobase/handler/ha_innodb.cc

```diff
@@ -1052,8 +1115,20 @@ static SHOW_VAR innodb_status_variables[]= {
   (char*) &export_vars.innodb_system_rows_updated, SHOW_LONG},
   {"num_open_files",
   (char*) &export_vars.innodb_num_open_files,		  SHOW_LONG},
+  {"s_lock_os_waits",
+  (char*) &rw_lock_stats.rw_s_os_wait_count,		  SHOW_LONGLONG},
+  {"s_lock_spin_rounds",
+  (char*) &rw_lock_stats.rw_s_spin_round_count,	  SHOW_LONGLONG},
+  {"s_lock_spin_waits",
+  (char*) &rw_lock_stats.rw_s_spin_wait_count,	  SHOW_LONGLONG},
   {"truncated_status_writes",
   (char*) &export_vars.innodb_truncated_status_writes,	  SHOW_LONG},
+  {"x_lock_os_waits",
+  (char*) &rw_lock_stats.rw_x_os_wait_count,		  SHOW_LONGLONG},
+  {"x_lock_spin_rounds",
+  (char*) &rw_lock_stats.rw_x_spin_round_count,	  SHOW_LONGLONG},
+  {"x_lock_spin_waits",
+  (char*) &rw_lock_stats.rw_x_spin_wait_count,	  SHOW_LONGLONG},
```

Comment:
I do not think a direct binding of these partitioned counters will work as intended. `int64_counter_t` is split into 64 slots, and these might access the first slot.
Also, please use `SHOW_ULONGLONG` instead of `SHOW_LONGLONG`.